### ISSUE TITLE
Plus sign in gallery folder name -> 404 error

### ISSUE THREAD

**Issue by tomasKucera** [novafacile/novagallery#21]
When creating a gallery with "+" sign in its folder name (e.g. "Me+you"), the album page returns 404 error.

The error comes from Router.php class where urldecode is used on line 70:
`$path = urldecode($path);`

**Comment by david-novafacile** [comment]
Good point! That could be a challenge to find a solution to work with encoded urls and have a plus in folder name. We will try to fix it in the next major version.

**Comment by david-novafacile** [comment]
Will be fixed in v2

**Comment by david-novafacile** [comment]
Solved with novaGallery v2.

You can try the new release [novaGallery v2 Beta1](https://github.com/novafacile/novagallery/releases/tag/v2.0-beta1). Please note that this is currently still a beta version.
